pom.xml

```diff
@@ -169,6 +170,49 @@
                 </configuration>
                 -->
             </plugin>
+            <plugin>
+                <groupId>de.thetaphi</groupId>
+                <artifactId>forbiddenapis</artifactId>
+                <version>2.4.1</version>
+                <configuration>
+                    <!--
+                      if the used Java version is too new,
+                      don't fail, just do nothing:
+                    -->
+                    <failOnUnsupportedJava>false</failOnUnsupportedJava>
+                    <bundledSignatures>
+                        <!--
+                          This will automatically choose the right
+                          signatures based on 'maven.compiler.target':
+                        -->
+                        <bundledSignature>jdk-unsafe</bundledSignature>
+                        <bundledSignature>jdk-deprecated</bundledSignature>
+                        <!-- disallow undocumented classes like sun.misc.Unsafe: -->
+                        <bundledSignature>jdk-non-portable</bundledSignature>
+                    </bundledSignatures>
+                    <excludes>
+                        <!-- Excluded because of "Helper7#getBeanMemInfo" -->
+                        <exclude>
+                            com/graphhopper/util/Helper7.class
+                        </exclude>
+                        <exclude>
+                            com/graphhopper/storage/UnsafeDataAccess.class
+                        </exclude>
+                        <!-- Forbidden class/interface use: java.io.FileReader [Uses default charset] -->
+                        <exclude>
+                            com/graphhopper/json/JsonFeatureConverter.class
+                        </exclude>
+                        <!-- orbidden class/interface use: java.io.FileReader [Uses default charset] -->
+                        <exclude>
+                            com/graphhopper/spatialrules/SpatialRuleLookupHelper.class
```

Comment:
These two use the FileReader. From the Javadoc:

```
The constructors of this class assume that the default character encoding and the default byte-buffer size are appropriate.  To specify these values yourself, construct an InputStreamReader on a FileInputStream.
```

I would fix this, if we decide to use PolicemanTools :).

Reply:
Ok, I fixed the FileReader issue as well.